### PR TITLE
.github: cancel workflow actions on forced push

### PR DESCRIPTION
**Pull request by peterbarker**
Just throwing stuff and seeing what sticks.

### CODE REVIEW

**Comment by peterbarker** [comment]
Based on https://github.com/marketplace/actions/cancel-workflow-action#advanced-pull-requests-from-forks

File: .github/workflows/cancel.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+name: Cancel
+on:
+  workflow_run:
+    workflows: [
+        "Cygwin Build",
+        "Notify users based on issue labels",
```
Comment (bugobliterator) [inline .github/workflows/cancel.yml:6]: This is not triggered with push, but with adding label. Don't think should be on this list.

**Review by bugobliterator — CHANGES_REQUESTED** [review]

**Comment by khancyr** [comment]
I am working on a PR with github action concurrency

**Comment by khancyr** [comment]
https://github.com/ArduPilot/ardupilot/pull/17847

**Comment by peterbarker** [comment]
> I am working on a PR with github action concurrency

Does it work?

**Comment by khancyr** [comment]
it seems working now !

**Comment by peterbarker** [comment]
@khancyr great!  I'll kill this one then :-)

**Pull request closed without merging**